src/Microsoft.AspNetCore.Razor.Sdk/SDK/Sdk.targets

```diff
@@ -0,0 +1,20 @@
+<!--
+***********************************************************************************************
+Sdk.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="$(_MicrosoftAspNetCoreRazorSdkCurrentVersionPath)" 
+    Condition="'$(_MicrosoftAspNetCoreRazorSdkCurrentVersionPath)' != '' AND Exists('$(_MicrosoftAspNetCoreRazorSdkCurrentVersionPath)')" />
```

Comment:
My goal here was to have a way to override the targets in the SDK with the targets in the package. The props file in package sets up this property to point to the target next to it and we use it. Let me know if there's a better way to do this, I couldn't find good examples for this.

Reply:
> My goal here was to have a way to override the targets in the SDK with the targets in the package

Outside of testing scenarios, I don't understand why this is a goal.

The intention is that the SDK provides 60% of the functionality, and the packages provide the other 40%.

Reply:
Testing - If we get an implicit reference to Razor.Sdk via Web.Sdk (❓) this would be a way to override it in our test projects.

OOB solutions - Akin to how Roslyn's Microsoft.NET.Compilers package, we could point users to this package if we want them to try our new SDK features or get bug fixes. 

Theoretically just adding the package reference would suffice since the targets from package would stomp over the targets from the SDK, this just seemed more predictable.